spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Wondering if these annotations should be in a separate maven module so that other libraries can include them without all of sleuth being required